lxd/container_lxc.go

```diff
@@ -7126,6 +7128,14 @@ func (c *containerLXC) IsPrivileged() bool {
 	return shared.IsTrue(c.expandedConfig["security.privileged"])
 }
 
+func (c *containerLXC) IsDevLxd() bool {
```

Comment:
The name of the function is misleading. `IsDevLxd()` makes it sound like it detects whether a given entity is devlxd but what it actually does is to see whether `devlxd` is supposed to be enabled in the container. So I'd say this should be called something like `EnableDevLxd()`.